sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -40,9 +40,17 @@
                 " /></v-col
           ></v-row>
         </content-group>
-        <v-divider></v-divider>
+        <v-divider
```

Comment:
may be we can wrap a template and add this duplicated logic in the template

Reply:
I don't think we should change the template for this.

Reply:
ok. Got it. Will do it.

Reply:
Done.